src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46079/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46079/project/issues?id=sample&pullRequest=6988&issues=AWleMlZaEo9FcZJAz3w0&open=AWleMlZaEo9FcZJAz3w0)

[ISSUE_KEY]: AWleMlZaEo9FcZJAz3w0